logstash-output-example.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name = 'logstash-output-example'
-  s.version         = '1.0.1'
+  s.version         = '2.0.2'
```

Comment:
@jakelandis I wasn't sure what to do here because the version in the gemspec did not AT ALL match the descriptions in the changelog.

Reply:
We don't release this as a gem, so it technically (in the literal sense) doesn't matter, but in the huh?? sense it does matter. This looks good and fixes the huh?? problem.